src/services/user.service.js

```diff
@@ -0,0 +1,52 @@
+const { User } = require('../models/User.model.js');
+
+const getUsers = async () => {
+  const users = await User.findAll();
+
+  return users;
+};
+
+const createUser = (name) => {
+  return User.create({ name });
+};
```

Comment:
The `createUser` function lacks validation for the `name` parameter. Consider adding validation to ensure that the name is a valid, non-empty string, similar to the `updateUser` function.